docs/quickstarts/aws.md

```diff
@@ -221,3 +273,5 @@ You can now start deploying your workloads on the cluster.
 
 For more information on installing supported Lokomotive components, you can visit the [component
 configuration references](../configuration-reference/components).
+
+[releases]: https://github.com/kinvolk/lokomotive/releases
```

Comment:
this is used only once, is it worth putting it here?

Reply:
It's not worth putting it here but I thought as we were using it in Packet how to guide, we could use the same format. I can remove it if not necessary